rule-packs/gcp.json

```diff
@@ -0,0 +1,134 @@
+[
+    {
+      "name": "gcp-personal-account-access",
+      "description": "Finds login credentials tied to personal accounts (as opposed to corporate credentials).",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "find google_user with email !$='@{{domain}'",
+          "version": "v1"
+        }
+      ]
+    },
+    {
+      "name": "gcp-managed-service-account-keys",
+      "description": "Finds user managed serice accounts with user managed keys (as opposed to system managed keys).",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "find google_iam_service_account_key with type!='SYSTEM_MANAGED'",
+          "version": "v1"
+        }
+      ]
+    },
+    {
+        "name": "gcp-service-account-admin-privilege",
```

Comment:
This may be best as a medium or info alert - I think the default of High might be unnecesary.